ecowitt2mqtt/main.py

```diff
@@ -113,6 +115,10 @@ def main():
 
     LOGGER.debug("Using arguments: %s", args)
 
+    # workaround for windows python-3.8.0a4 NotImplementedError
```

Comment:
This is a wholly separate issue (Windows support) than what the PR states and should be in its own PR.